### PR TITLE
add FAQ how to create clusters

### PR DESCRIPTION
**Pull request by medyagh**
based on this survery response:

```
6/29/2021 6:20:02 | local testing |   | everything i tried was good | Linux | 3 | create more than one cluster
```

### CODE REVIEW

File: site/content/en/docs/faq/_index.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How to create an multiple clusters using minikube ?
```
Comment (spowelljr) [inline site/content/en/docs/faq/_index.md:21]: I would move away from the word `multiple` so people don't get confused with multinode.

Maybe: `How can I create additional clusters?`

Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:21]: How can I create multiple clusters with minikube?

File: site/content/en/docs/faq/_index.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ Example:
...
+
+By default `minikube start` creates a cluster named minikube, if you would like to create another cluster or change the name.
```
Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:23]: By default, `minikube start` creates a cluster named `minikube`. If you would like to create a different cluster or change its name, you can use the `--profile` (or `-p`) flag, which will create a cluster with the specified name.

File: site/content/en/docs/faq/_index.md [hunk 3]
```diff
@@ -18,6 +18,20 @@ Example:
...
+you could use the `-p` or `--profile` flag. which will create a clustter with the specific name.
+
+minikube profiles are meant to be isolated from each other, and each one could have their own settings or driver. if you want to create one cluster with multiple nodes the multi-node feature. instead
```
Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:26]: minikube profiles are meant to be isolated from one another, with their own settings and driver. If you want to create a single cluster with multiple nodes, try the [multi-node feature](link to multinode docs) instead.

File: site/content/en/docs/faq/_index.md [hunk 4]
```diff
@@ -18,6 +18,20 @@ Example:
...
+
+To see list of your current clisters, try
```
Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:28]: To see the list of your current cluster, run:

**Review by sharifelgamal — CHANGES_REQUESTED** [review]

File: site/content/en/docs/faq/_index.md [hunk 5]
```diff
@@ -18,6 +18,19 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How can I create more than one cluster with minikube?
+
+By default, minikube start creates a cluster named minikube. If you would like to create a different cluster or change its name, you can use the --profile (or -p) flag, which will create a cluster with the specified name.
```
Comment (sharifelgamal) [inline site/content/en/docs/faq/_index.md:23]: ad the backtick `minikube start`, `minikube` (the cluster name), `--profile` and `-p`

Comment (medyagh) [reply site/content/en/docs/faq/_index.md]: done

File: site/content/en/docs/faq/_index.md [hunk 6]
```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How can I create more than one cluster with minikube?
+
+By default, `minikube start` creates a cluster named "minikube". If you would like to create a different cluster or change its name, you can use the `--profile` (or `-p`) flag, which will create a cluster with the specified name. and you could have multiple clusters on the same machine.
+
+To see list of your current clisters, run:
```
Comment (andriyDev) [inline site/content/en/docs/faq/_index.md]: "To see a list of all existing clusters, run:"

File: site/content/en/docs/faq/_index.md [hunk 7]
```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How can I create more than one cluster with minikube?
+
+By default, `minikube start` creates a cluster named "minikube". If you would like to create a different cluster or change its name, you can use the `--profile` (or `-p`) flag, which will create a cluster with the specified name. and you could have multiple clusters on the same machine.
```
Comment (andriyDev) [inline site/content/en/docs/faq/_index.md:23]: "You can have multiple clusters on the same machine."

File: site/content/en/docs/faq/_index.md [hunk 8]
```diff
@@ -18,6 +18,20 @@ Example:
...
+
+To see list of your current clisters, run: 
+```
+minikube profile list
+```
+
+minikube profiles are meant to be isolated from one another, with their own settings and driver. If you want to create a single cluster with multiple nodes, try the [multi-node feature]({{< ref "docs/tutorials/multi_node/" >}}) instead.
```
Comment (andriyDev) [inline site/content/en/docs/faq/_index.md:30]: "... and drivers"

File: site/content/en/docs/faq/_index.md [hunk 9]
```diff
@@ -18,6 +18,20 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How can I create more than one cluster with minikube?
+
+By default, `minikube start` creates a cluster named "minikube". If you would like to create a different cluster or change its name, you can use the `--profile` (or `-p`) flag, which will create a cluster with the specified name. and you could have multiple clusters on the same machine.
+
+To see the list of your current clusters, run:
```
Comment (andriyDev) [inline site/content/en/docs/faq/_index.md:25]: "To see a list of all existing minikube clusters, run:"

**Review by andriyDev — CHANGES_REQUESTED** [review]

**Review by sharifelgamal — APPROVED** [review]

**Pull request merged**